python/ray/autoscaler/_private/gcp/config.py

```diff
@@ -67,7 +72,11 @@ def tpu_accelerator_config_to_type(accelerator_config: dict) -> str:
     # Reduce e.g. "2x2x2" to 8
     chip_dimensions = [int(chip_count) for chip_count in topology.split("x")]
     num_chips = reduce(lambda x, y: x * y, chip_dimensions)
-    num_cores = num_chips * 2
+    num_cores = num_chips * DEFAULT_TPU_CORES_PER_CHIP
+
+    if generation == "v5lite_pod":
```

Comment:
Why do we use `V5LITE_POD` in `accelerator_config["type"]` instead of `V5LITEPOD`? In this PR, we use both. Is it possible to make them consistent?

Reply:
Unfortunately I think this comes from an internal naming consistency in Google internal proto code. I am not sure how easy it would be to modify this.

@allenwang28

Reply:
Thank you both for the explanation! Maybe we should at least add some comments about when to use `V5LITE_POD` and when to use `V5LITEPOD`.

Reply:
Added a comment.